qemu/tests/create_large_raw_img.py

```diff
@@ -38,11 +40,14 @@ def run(test, params, env):
     try:
         large.create(large.params)
     except TestError as err:
-        if params["err_info"] not in str(err):
+        for info in err_info:
+            if info in six.text_type(err):
```

Comment:
If you meet the unicode compatible with both python 2 and 3
```suggestion
            if info in str(err):
```
FYI:
```
>>> a =1 / 0
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ZeroDivisionError: integer division or modulo by zero
```
python2
```
>>> try:
...     a = 1 / 0 
... except ZeroDivisionError as e:
...     if 'divisio' in str(e):
...         print('catch')
... 
catch
```
python3
```
>>> try:
...     a = 1 / 0
... except ZeroDivisionError as e:
...     if 'divisio' in str(e):
...         print('catch')
... 
catch
>>> 
```

Reply:
I see, str(err) works well. We suggest to use six.text_type(err) instead of str(err).

Python 2.7.5 (default, Jul 16 2018, 19:52:45) 
[GCC 4.8.5 20150623 (Red Hat 4.8.5-36)] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import six
>>> try:
...     a = 1 /0
... except ZeroDivisionError as e:
...     if 'divisio' in six.text_type(e):
...         print("catch")
... 
catch
>>>

Python 3.6.8 (default, Jun 26 2020, 12:10:09) 
[GCC 8.3.1 20191121 (Red Hat 8.3.1-5)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import six
>>> try:
...     a = 1 / 0
... except ZeroDivisionError as e:
...     if 'divisio' in six.text_type(e):
...         print("catch")
... 
catch